conda_smithy/configure_feedstock.py

```diff
@@ -441,18 +441,21 @@ def dump_subspace_config_files(
             f"{platform}_{arch}",
             package_key(config, top_level_loop_vars, metas[0].config.subdir),
         )
+        short_config_name = config_name
+        if len(short_config_name) >= 49:
+            h = hashlib.sha256(config_name.encode("utf-8")).hexdigest()[:10]
+            short_config_name = config_name[:35] + "_h" + h
+        if len(config_name + ".yaml") >= 250:
```

Comment:
Usually 255 is a hard limit. Sometimes, e.g., on an overlay filesystem some space for additional characters (`.wh.`) is reserved too. Going for <250 should be fine.